packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -30,6 +30,7 @@ export class Runtime {
   public static readonly NodeJS43 =     new Runtime('nodejs4.3',      RuntimeFamily.NodeJS, { supportsInlineCode: true });
   public static readonly NodeJS610 =    new Runtime('nodejs6.10',     RuntimeFamily.NodeJS, { supportsInlineCode: true });
   public static readonly NodeJS810 =    new Runtime('nodejs8.10',     RuntimeFamily.NodeJS, { supportsInlineCode: true });
+  public static readonly NodeJS10x =    new Runtime('nodejs10.x',     RuntimeFamily.NodeJS, { supportsInlineCode: true });
```

Comment:
IIUC `supportsInlineCode` should be `false` here, no?

Reply:
For future reference, this is because CloudFormation has hardcoded an input validation that hasn't been updated to know about the new version yet.